generated/python/grpc-google-cloud-speech-v1beta1/setup.py

```diff
@@ -0,0 +1,43 @@
+"""A setup module for the GRPC google-cloud-speech service.
+
+See:
+https://packaging.python.org/en/latest/distributing.html
+https://github.com/pypa/sampleproject
+"""
+
+import setuptools
+
+from setuptools import setup, find_packages
+
+install_requires = [
+  'oauth2client>=1.4.11
```

Comment:
Missing closing quote and comma. Anyway, can you change this to `>=3.0.0, <4.0.0dev`?